### PR TITLE
Update RELEASE.md to add entry for modular file system support (s3 and hdfs)

### PR DESCRIPTION
**Pull request by yongtang**
This PR update RELEASE.md to add entry for modular file system support (s3 and hdfs).

Signed-off-by: Yong Tang <yong.tang.github@outlook.com>

### CODE REVIEW

**Review by kvignesh1420 — COMMENTED** [review]
Thanks @yongtang

File: RELEASE.md [hunk 1]
```diff
@@ -209,6 +209,11 @@ This release contains contributions from many people at Google, as well as:
         service workers to preserve existing behavior. The default value is
         `"AUTO"`.
 
+*   Modular File System Migration
+    * S3 and HDFS file system supports have been migrated to modular file systems
+      and is now available in https://github.com/tensorflow/io. The tensorflow-io
+      python package can be installed for S3 and HDFS support with tensorflow.
```
Comment (kvignesh1420) [inline RELEASE.md:215]: @yongtang I think it should be:

"The tensorflow-io python package should be installed for S3 and HDFS support with tensorflow"

Comment (yongtang) [reply RELEASE.md]: @kvignesh1420  The PR has been updated.

**Review by kvignesh1420 — APPROVED** [review]

File: RELEASE.md [hunk 2]
```diff
@@ -209,6 +209,11 @@ This release contains contributions from many people at Google, as well as:
         service workers to preserve existing behavior. The default value is
         `"AUTO"`.
 
+*   Modular File System Migration
+    * S3 and HDFS file system supports have been migrated to modular file systems
+      and is now available in https://github.com/tensorflow/io. The tensorflow-io
+      python package should be installed for S3 and HDFS support with tensorflow.
+
```
Comment (mihaimaruseac) [inline RELEASE.md:216]: I'd move this in the breaking changes section

Comment (yongtang) [reply RELEASE.md]: Thanks @mihaimaruseac . The PR has been updated.

**Review by mihaimaruseac — APPROVED** [review]

**Review by gbaned — APPROVED** [review]

**Pull request merged**